recipes/newrelic/mongodb/rhel.yml

```diff
@@ -0,0 +1,170 @@
+
+name: mongodb-open-source-integration
+displayName: MongoDB Open Source Integration
+description: New Relic install recipe for default MongoDB Open Source on-host integration (via Infra-Agent)
+repository: https://github.com/newrelic/nri-mongodb
+
+installTargets:
+  - type: host
+    os: linux
+    platform: "amazon"
+    platformVersion: "2"
+  - type: host
+    os: linux
+    platform: "redhat"
+  - type: host
+    os: linux
+    platform: "centos"
+
+keywords:
+  - Infrastructure
+  - Integration
+  - mongodb
+
+processMatch:
+  - mongodb
+  - mongod
+
+logMatch:
+  - name: MongoDB log
+    file: /var/log/mongodb/mongod.log
+
+validationNrql: "SELECT count(*) from MongoDatabaseSample where hostname like '{{.HOSTNAME}}%' FACET entityGuid SINCE 10 minutes ago"
+
+inputVars:
+  - name: "NR_CLI_USERNAME"
+    prompt: "Please enter your MongoDb username below."
+  - name: "NR_CLI_CLUSTERNAME"
+    prompt: "Please enter your MongoDb culster name below."
+  - name: "NR_CLI_PASSWORD"
+    prompt: "Please enter your MongoDb password below."
+    secret: true
+  - name: "NR_CLI_AUTH"
+    prompt: "Please confirm auth source for MongoDb below."
+    default: "admin"
+  - name: "NR_CLI_HOSTNAME"
+    prompt: "Please enter your MongoDb hostname below. If none is provided the default value: localhost will be used."
+    default: "localhost"
+  - name: "NR_CLI_PORT"
+    prompt: "Please enter your MonogDb PORT below. If none is provided the default value: 27017 will be used."
+    default: 27017
+  - name: "NR_CLI_SSL"
+    prompt: "Are you using SSL? (y/n)"
+  - name: "NR_CLI_CERT_AUTH_FILE"
+    prompt: "Please enter the path to your Certificate Authority file."
+  - name: "NR_CLI_SKIP_SSL_VERIFY"
+    prompt: "Would you like to verify the certificate sent by the host? (y/n)"
+  - name: "NR_CLI_CLIENT_CERT_FILE"
+    prompt: "Please enter the path to your Client Certificate file, if required."
+  - name: "NR_CLI_CERT_PASSPHRASE"
+    prompt: "Please enter your passphrase to decrypt the PEM key file, if required."
```

Comment:
Should review these and align them with how our other prompts look (maybe check out the [Elasticsearch](https://github.com/newrelic/open-install-library/blob/main/recipes/newrelic/elasticsearch/rhel.yml#L36-L56) recipe for example)